src/Common/src/System/Net/Security/Unix/SafeDeleteNegoContext.cs

```diff
@@ -78,6 +88,12 @@ protected override void Dispose(bool disposing)
                     _targetName.Dispose();
                     _targetName = null;
                 }
+
+                if (_acceptorCredential != null)
```

Comment:
I know you've already merged this and there's existing code with this pattern, but this isn't thread safe. If Dispose is called on two threads at the same time, you can get a null reference exception. You could simply replace this with:
```c#
_acceptorCredential?.Dispose();
_acceptorCredential = null;
```
The generated IL is thread safe when using the `?.` operator.